lib/rendition.js

```diff
@@ -236,6 +220,35 @@ class Rendition {
             console.log("could not determine encoding/charset");
         }
 
+        try {
+            meta[Metadata.REPO_SIZE] = this.size();
+        } catch (err) {
+            console.log("could not determine file size:", err);
+        }
+
+        try {
+            const dimensions = await imageSize(this.path);
```

Comment:
The library we use here, [image-size]() has some limitations that can lead to width/height not being found:
- some JPEGs are [not supported](https://github.com/image-size/image-size/blob/main/lib/types/jpg.ts#L1-L4);
- bigendian TIFF files [are currently not supported](https://github.com/image-size/image-size/blob/main/lib/types/tiff.ts#L2).